src/components/notebook-utils.js

```diff
@@ -76,7 +76,7 @@ export const tools = {
   Jupyter: { label: 'Jupyter', ext: 'ipynb', imageIds: ['terra-jupyter-bioconductor', 'terra-jupyter-bioconductor_legacy', 'terra-jupyter-hail', 'terra-jupyter-python', 'terra-jupyter-gatk', 'Pegasus', 'terra-jupyter-gatk_legacy'], defaultImageId: 'terra-jupyter-gatk' },
   jupyterTerminal: { label: 'terminal' },
   spark: { label: 'spark' },
-  galaxy: { label: 'galaxy', appType: 'GALAXY' },
+  galaxy: { label: 'Galaxy', appType: 'GALAXY' },
   cromwell: { label: 'cromwell', appType: 'CROMWELL', isAppHidden: !isCromwellAppVisible(), isPauseUnsupported: true }
```

Comment:
```suggestion
  cromwell: { label: 'Cromwell', appType: 'CROMWELL', isAppHidden: !isCromwellAppVisible(), isPauseUnsupported: true }
```

should fix the issue mentioned on this [comment](https://github.com/DataBiosphere/terra-ui/pull/2896#issuecomment-1074444223).

Reply:
updated